conversions/binary_to_octal.dart

```diff
@@ -0,0 +1,42 @@
+//Author:VishnuPothan
+//Binary number to octal number conversion
+void main() {
+  print(binary_to_octal("-1111")); // -17
+  print(binary_to_octal("101011")); // 53
+  print(binary_to_octal("1011a01")); // error
+  print(binary_to_octal("")); // error
+}
+
+String binary_to_octal(String bin_string) {
+  bin_string = bin_string.trim();
+  if (bin_string == null || bin_string == "") {
+    return ("An empty value was passed to the function");
```

Comment:
Throwing exception is more intuitive here
```suggestion
    throw new FormatException("An empty value was passed to the function");
```